website/docs/getting-started/for-engineers.md

```diff
@@ -70,37 +70,157 @@ app.import('node_modules/@hashicorp/design-system-components/dist/styles/@hashic
+
+Because the `Hds::Icon` component is part of the HDS components, you have to install the corresponding package:
+
+```bash
+yarn add @hashicorp/design-system-components
+```
+
+and then use the component in your code like this:
+
+```handlebars{data-execute=false}
+<Hds::Icon @name="info" />
+```
+
+For details about how this component should be used and its API see [the component documentation page](/components/icon).
+
+#### Using the `@hashicorp/ember-flight-icons` addon <Doc::Badge @type="warning" @size="large">Deprecated</Doc::Badge>
 
-### Install icons package
+!!! Warning
+
+This approach is now deprecated. Use the `Hds::Icon` instead.
+
+!!!
 
-#### Ember application
+The `ember-flight-icons` package is an Ember addon that provides a standalone `FlightIcon` component that can be used to render an icon as an `<svg>` HTML element.
+
+Since it's an independent package, you have to install it first:
 
 ```bash
 yarn add @hashicorp/ember-flight-icons
 ```
 
-#### React application
+and then use the component in your code like this:
+
+```handlebars{data-execute=false}
+<FlightIcon @name="info" />
+```
+
+The [API of the component](/components/icon?tab=code#component-api) is the same as the `Hds::Icon` one.
+
+#### Deferred loading
+
+In both approaches, the SVG sprite will be injected by default into your application's `index.html` file. If you would like this to happen later as part of your app bundle you can set the `lazyEmbed` flag to `true` in the `emberFlightIcons` object in your app's `config/environment.js` file:
+
+```js
+module.exports = function(environment) {
+  const ENV = {
+    // your other config
+    ...
+    emberFlightIcons: {
+      lazyEmbed: true,
+    },
+  };
+}
+```
+
+For more information on why this may be helpful in certain scenarios, see [DS-049 - Improve Ember Flight Icons Loading Performance](https://go.hashi.co/rfc/ds-049).
+
+#### Ember test selectors
+
+Both the `Hds::Icon` and the `FlightIcon` components expose a `data-test-icon` helper. For this reason we recommend installing [`ember-test-selectors`](https://github.com/simplabs/ember-test-selectors) which strips out all `data-test-*` attributes for production builds.
+
+#### Using the icons without importing the whole components package
+
+If you want to use the Flight icons without installing the whole `@hashicorp/design-system-components` package, you have to use the `@hashicorp/flight-icons` to import the SVG sprite, and then you will have to build your own Ember component that renders the icons as an `<svg>` HTML element.
+
+You can copy the code for the `Hds::Icon` in your codebase, or you can [take inspiration from this PR](https://github.com/hashicorp/design-system-metrics/pull/23) to build your own component.
+
+### React applications
+
+To add icons to a React application, you need to install the `@hashicorp/flight-icons` package:
 
 ```bash
-yarn add @hashicorp/flight-icons
+yarn install @hashicorp/flight-icons
```

Comment:
blocker: `yarn add` adds new packages; `yarn install` doesn't take arguments and installs the existing packages

Reply:
Yep!

Reply:
```suggestion
yarn add @hashicorp/flight-icons
```